mappings/case-list.json

```diff
@@ -15,6 +15,7 @@
     },
     "status": 200,
     "jsonBody": {
+      "lastUpdated": "2019-12-03T08:30:00",
```

Comment:
Actually, might be better to change this to use the current date:

"lastUpdated": "{{now format='yyyy-MM-dd'}}T08:30:00",

Reply:
Yeah I did consider this but I'm kinda wary as it means both the data and the test will be dynamic and therefore non-deterministic. I agree it's nice to have it show realistic data though, will do as you suggest and we can review it if it causes any problems, which tbf I doubt it will.

Reply:
...strictly speaking it is deterministic actually coming to think of it so even less of a problem. Slight chance of edge cases causing arbitrary failures but unlikely.